docs/docs/20-usage/20-pipeline-syntax.md

```diff
@@ -727,6 +727,15 @@ pipeline:
   ...
 ```
 
+## `skip_clone`
+
+If you don't specify (a) clone step(s), a default one is added.
+So if you like to not have one at all, just specify:
```

Comment:
```suggestion
If you don't specify clone steps, a default one is added.
So if you like to not have one at all, just specify:
```
?

Reply:
```suggestion
By default Woodpecker is automatically adding a clone step. This clone step can be configured by the [clone](#clone) property. If you do not need a `clone` step at all you can skip it using:
```

Reply:
@qwerty287 As far as I know you could add steps to the pipeline area with one of the trusted clone images which would get the netrc credentials passed which is possible since #1352 (pretty cool IMO and somehow makes the clone area obsolete to be fair)